Monitors section home page (#4835)
DataDog/documentation: content/en/monitors/monitor_status.md

## Header
On the left, the header contains the monitor's status, time of status, and monitor title.

On the right, the header contains **Mute**, **Resolve**, and the **Settings** cog.

### Mute
Choose to mute a monitor directly on its status page. Use the **Scope** field to narrow your downtime. Refer to [Downtimes][2] to learn how to mute multiple scopes or multiple monitors at the same time.

**Note**: Muting or unmuting a monitor with the UI deletes all scheduled downtimes associated with that monitor.

### Resolve
If your monitor is in an alert state, the **Resolve** button is visible. Use this button to resolve your monitor manually.

The monitor `resolve` function is artificially switching the monitor status to `OK` for its next evaluation. The next monitor evaluation is performed normally on the data the monitor is based on.

If a monitor is alerting because its current data corresponds to its `ALERT` state, `resolve` has the monitor follow the state switch `ALERT -> OK -> ALERT`. Thus, it's not appropriate for acknowledging that you have seen the alert or telling Datadog to ignore the alert.

> Manually `resolve`-ing a monitor is appropriate for cases where data is reported intermittently: after triggering an alert, the monitor doesn't receive further data so it can no longer evaluate alerting conditions and recover to the `OK` state. In that case the `resolve` function or the `Automatically resolve monitor after X hours` switches the monitor back to `OK` state.

Reviewer: ```suggestion
Manually `resolve`-ing a monitor is appropriate for cases where data is reported intermittently: after triggering an alert, the monitor doesn't receive further data so it can no longer evaluate alerting conditions and recover to the `OK` state. In that case, the `resolve` function or the `Automatically resolve monitor after X hours` switches the monitor back to `OK` state.
```

> **Typical use case**: A monitor based on error metrics that are not generated when there are no errors (e.g. `aws.elb.httpcode_elb_5xx`, or any DogStatsD counter in your code reporting an error _only when there is an error_).

Reviewer: ```suggestion
**Typical use case**: A monitor based on error metrics that are not generated when there are no errors (`aws.elb.httpcode_elb_5xx`, or any DogStatsD counter in your code reporting an error _only when there is an error_).
```

|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| [Edit][1] | Edit the current monitor. More information is available in [Monitor Types][1].                                                                                                                                                             |
| Clone     | Make a copy of the current monitor.                                                                                                                                                                                                        |
| Export    | Export the JSON configuration for the current monitor. This option is also available when [creating your monitor][1]. If you manage and deploy monitors programmatically, it's easier to define the monitor in the UI and export the JSON. |

> | Delete    | Delete the current monitor. You will be prompted to confirm deletion.                                                                                                                                                                      |

Reviewer: ```suggestion
| Delete    | Delete the current monitor. You will be prompted to confirm the deletion.                                                                                                                                                                      |
```

* The status of your monitor
* The monitor creator
* The monitor ID ([for the monitor API][3])
* Tags attached to your monitor. *Edit the tag list by clicking on the pencil icon*.
* The monitor [query][4]
* The monitor message

Use the *cog* icon in the upper right corner of the page to [edit][1] your monitor properties.

## Status and History

> The *Status and History* section reflect the query and state changes over time, while the **Evaluation Graph** represents the exact query behavior within the timeframe bracket *on the history graph*. The Evaluation Graph has a fixed zoomed window that corresponds to your monitor evaluation timesteps, to ensure the displayed points are not [incorrectly aggregated][5]. Slide this bracket over the timeline in order to view previous monitor evaluation results:

Reviewer: ```suggestion
The *Status and History* section reflect the query and state changes over time, while the **Evaluation Graph** represents the exact query behavior within the timeframe bracket *on the history graph*. The Evaluation Graph has a fixed zoomed window that corresponds to your monitor evaluation timesteps, to ensure the displayed points [aggregated correctly][5]. Slide this bracket over the timeline to view previous monitor evaluation results:
```

> 

Reviewer: I can't make this as a suggestion since it's not a change you made, but hostname should be one word.
